src/collectors/portstat/portstat.py

```diff
@@ -0,0 +1,69 @@
+
+"""
+The PortStatCollector collects metrics about ports listed in config file.
+
+##### Dependencies
+
+* psutil
+
+"""
+
+from collections import Counter
+
+import psutil
+import diamond.collector
+
+
+def get_port_stats(port):
+    '''
+    Iterate over connections and counts states for specified port
```

Comment:
language: "*Iterates* over connections" or "Iterate over connections and *count*"